content/docs/11.migration-guide/input-files.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Deprecation of LocalFiles and outputDir
+icon: /docs/icons/migration-guide.svg
+release: 0.17.0
+---
+
+`LocalFiles` and `outputDir` were depreciated in Kestra 0.17.0. Here is why:
```

Comment:
```suggestion
Check out this guide to migrate from `LocalFiles` and `outputDir` to `inputFiles` and `outputFiles`.

## Overview
The `LocalFiles` and `outputDir` are deprecated due to overlapping functionality that already exists using `inputFiles` and `outputFiles` on the `WorkingDirectory` and [script](https://kestra.io/docs/developer-guide/scripts) tasks.
```

keep in mind the first sentence must be short to be displayed on the card preview + social card